apps/cowswap-frontend/src/modules/orderProgressBar/hooks/useOrderProgressBarProps.ts

```diff
@@ -290,7 +290,7 @@ function useCountdownStartUpdater(
 
   useEffect(() => {
     if (shouldDisableCountdown) {
-      if (countdown) {
+      if (countdown != null) {
```

Comment:
Any reason not to use strict comparison?
`!==` instead of `!=`

Reply:
> We switched to countdown != null so that both null and undefined disable the countdown while allowing 0 to pass through. A strict check (!== null) would still let undefined slip by, so we’d need the extra countdown !== undefined clause, hence the loose null check here.